graph/euler's_tour.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+Problem Description:
+Euler tour is defined as a way of traversing tree such that each vertex is added
+to the tour when we visit it (either moving down from parent vertex or returning
+from child vertex). We start from root and reach back to root after visiting all
+vertices. It requires exactly 2*n-1 vertices to store Euler tour.
+
+Test Case :
+input:
+4
+1 2
+2 3
+2 4
+
+output:
+1 2 3 2 4 2 1
+*/
```

Comment:
**
 * @file
 * @brief Add one line description here. Should contain a Wikipedia
 * link or another source explaining the algorithm/implementation.
 * @details
 * This is a multi-line
 * description containing links, references,
 * math equations, etc.
 * @author [Name](https://github.com/handle)
 * @see related_file.cpp, another_file.cpp
 */

Reply:
https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md
Please read this.

Reply:
resolved. pls check